packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -16,46 +17,46 @@ describe('Persona', () => {
 
     it('renders with expected initialsColor if none was provided', () => {
       const wrapper = shallow(<Persona primaryText='Kat Larrson' />);
-      let result = wrapper.find('.ms-Persona-initials--red');
+      let result = wrapper.find('.' + styles.isRootInitialsRed);
```

Comment:
this is not a state but a variant, so use rootInitialsRed, or rootInitialsAreRed. Also 'is initials' isn't proper grammar.